app/src/main/java/com/example/myplaylistmaker/SearchActivity.kt

```diff
@@ -98,17 +98,20 @@ class SearchActivity : AppCompatActivity() {
         }
 
         inputEditText.setOnFocusChangeListener { view, hasFocus ->
-            if (hasFocus && inputEditText.text.isEmpty() ) {
+            if (hasFocus && inputEditText.text.isEmpty() && history.getHistoryTrackList() != ArrayList<Track>() ) {
```

Comment:
🍏 Если нужна проверка на то, пустой ли список, то лучше использовать специальный метод коллекции
```suggestion
            if (hasFocus && inputEditText.text.isEmpty() && history.getHistoryTrackList().isNotEmpty()) {
```